libgeopm/src/geopmbench_main.cpp

```diff
@@ -110,10 +110,14 @@ static int main_imp(int argc, char **argv)
 "\n";
 
     const int ERROR_HELP = -4096;
+    const geopm::GEOPMBenchConfig &config = geopm::geopmbench_config();
+    bool is_mpi_enabled = config.is_mpi_enabled();
 #ifdef GEOPM_ENABLE_MPI
```

Comment:
```suggestion
#ifdef GEOPM_ENABLE_MPI
    const geopm::GEOPMBenchConfig &config = geopm::geopmbench_config();
    bool is_mpi_enabled = config.is_mpi_enabled();

```

Reply:
Without this change i think we will have unused variable errors when mpi is disabled.